Alerts table/page updates for 8.7 (#3058)
elastic/security-docs: docs/detections/alerts-ui-manage.asciidoc

* Use the date and time filter to define a specific time range. By default, this filter is set to search the last 24 hours.

> * Visualize and group alerts by specific parameters in the visualization section. Use the buttons on the left to select a view type (*Summary*, *Trend*, *Counts*, or *Treemap*), and use the menus on the right to select the ECS fields used for grouping alerts. Refer to <<visualize-alerts>> for more on each view type.

Reviewer: ```suggestion
* Visualize and group alerts by specific fields in the visualization section. Use the buttons on the left to select a view type (*Summary*, *Trend*, *Counts*, or *Treemap*), and use the menus on the right to select the ECS fields used for grouping alerts. Refer to <<visualize-alerts>> for more on each view type.
```
I don't think this is an accurate use of '[parameter](https://www.techopedia.com/definition/3725/parameter-param)',

Author: It was intended in a more non-technical sense, but since these are technical docs, I agree accuracy is better here.

* View detection alerts generated by a specific rule. Go to *Manage* -> *Rules*, then select a rule name in the table. The rule details page displays a comprehensive view of the rule's settings, and the Alerts table under the Trend histogram displays the alerts associated with the rule, including alerts from any previous or deleted revision of that rule.

[float]
[[group-alerts]]
=== Group alerts

> beta:[] You can group alerts by rule name, host name, user name, source IP address, or any other field. Select *Group alerts by*, then select an option or *Custom field* to specify a different field.

Reviewer 2: Is this `beta:[]` tag correct? It looks weird rendered: 

<img width="640" alt="Screenshot 2023-03-28 at 8 07 31 AM" src="https://user-images.githubusercontent.com/6935300/228265181-4fdbc3d6-216c-47cc-af23-b365ce0da7cd.png">

[role="screenshot"]
image::images/group-alerts.png[Alerts table with Group alerts by drop-down]

> Each group displays information such as the alerts' severity and how many users, hosts, and alerts are in the group. The type of information displayed varies depending on the field used for grouping.

Reviewer 3: ```suggestion
Each group displays information such as the alerts' severity and how many users, hosts, and alerts are in the group. The type of information displayed varies depending on the selected field.
```

To interact with grouped alerts:

> * Select *Take actions* to perform a bulk action on all alerts in a group, such as <<detection-alert-status,changing their status>>.

Reviewer 3: I'll leave this comment once for wherever we mention `Select Take actions`.  By UX standards, it's technically a menu. 
```suggestion
* Select the *Take actions* menu to perform a bulk action on all alerts in a group, such as <<detection-alert-status,changing their status>>.
```

> * Click a group's expand icon (image:images/expand-icon-vertical-right.png[Grouped alerts expand icon,16,16]) to display alerts within that group. You can filter and customize this view like any other alerts table.

Reviewer 2: ```suggestion
* Click a group's expand icon (image:images/expand-icon-vertical-right.png[Grouped alerts expand icon,16,16]) or group name to display alerts within that group. You can filter and customize this view like any other alerts table.
```

You can click the carrot or the group name to expand.

> * Click a group's expand icon (image:images/expand-icon-vertical-right.png[Grouped alerts expand icon,16,16]) to display an alerts table within that group. You can filter and customize this view like any other alerts table.

Reviewer 4: Might be more useful to focus on what's being grouped instead of the view that's shown. 

```suggestion
* Click a group's expand icon (image:images/expand-icon-vertical-right.png[Grouped alerts expand icon,16,16]) to display alerts within that group. You can filter and customize this view like any other alerts table.
```

To change an alert's status, do one of the following:

* In the alert's row, click the *More actions* menu (*...*) in the Alerts table, then select a status.

> * In the Alerts table, select all the alerts you want to change, click on the drop-down at the upper-left above the table, and then select a status.

Reviewer 4: ```suggestion
* In the Alerts table, select the alerts you want to change, click on the drop-down at the upper-left above the table, and then select a status.
```

> * In the Alerts table, select all the alerts you want to change, click on the drop-down at the upper-left above the table, and then select a status.

Reviewer: ```suggestion
* In the Alerts table, select all the alerts you want to change, click *Selected _N_ alerts* at the upper-left above the table, and then select a status.
```
Not sure if X or N would be better, and the picture makes it pretty clear as-is, but I think naming the button couldn't hurt

Author: Good call, lowercase _x_ is the [preferred format](https://docs.elastic.dev/content-design/tech-docs/writing-about-ui#general-guidelines-for-writing-about-the-ui):
```suggestion
* In the Alerts table, select all the alerts you want to change, click *Selected _x_ alerts* at the upper-left above the table, and then select a status.
```

+
[role="screenshot"]
image::images/alert-change-status.png[Bulk action menu with multiple alerts selected, 225]

> * beta:[] To bulk-change the status of <<group-alerts,grouped alerts>>, select *Take actions* for the group, then select a status.

Reviewer 3: ```suggestion
* beta:[] To bulk-change the status of <<group-alerts,grouped alerts>>, select the *Take actions* menu for the group, then select a status.
```

> * In the alert details flyout, click *Take action* and select a status.

Reviewer: ```suggestion
* In an alert's details flyout, click *Take action* and select a status.
```
